src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/AzureRM.Storage.ps1

```diff
@@ -27,14 +27,19 @@ function New-AzureRmStorageAccount
     [string] [Parameter(Position=0, ValueFromPipelineByPropertyName=$true)] $ResourceGroupName,
     [string] [Parameter(Position=1, ValueFromPipelineByPropertyName=$true)][alias("StorageAccountName")] $Name,
 	[string] [Parameter(Position=2, ValueFromPipelineByPropertyName=$true)] $Location,
-	[string] [Parameter(Position=3, ValueFromPipelineByPropertyName=$true)] $Type)
+    [Microsoft.Azure.Management.Storage.Models.AccountType] [Parameter(Position=3, ValueFromPipelineByPropertyName=$true)] $Type)
   BEGIN { 
     $context = Get-Context
 	$client = Get-StorageClient $context
   }
   PROCESS {
     $createParms = New-Object -Type Microsoft.Azure.Management.Storage.Models.StorageAccountCreateParameters
-	$createParms.AccountType = [Microsoft.Azure.Management.Storage.Models.AccountType]::StandardLRS
+    if ($Type -eq $null)
+    {
+      $Type = [Microsoft.Azure.Management.Storage.Models.AccountType]::StandardLRS
+    }
+
+	$createParms.AccountType = $Type
```

Comment:
I don't think there is AccountType property in Microsoft.Azure.Management.Storage.Models.StorageAccountCreateParameters.  There is Sku property instead.